certmanager/ca.go

```diff
@@ -103,7 +103,9 @@ func (ca IstioCA) Generate(name, namepsace string) (chain, key []byte) {
 		SignerCert:   ca.signingCert,
 		SignerPriv:   ca.signingKey,
 		IsCA:         false,
+		IsClient:     true,
```

Comment:
Do we need these options (IsClient and IsServer) any more? Since CA can simply generate a generic server used by server and client.